spec/active_record/comments_spec.rb

```diff
@@ -72,6 +72,26 @@ def capture_sql
       end
       sql.should == 'SELECT * FROM "users" WHERE "users"."id" = 1 /* xxx */'
     end
+
+    it "should be thread safe" do
+      res = []
+      ["xx", "yy"].map do |comment|
+        Thread.new do
+          sql = nil
+          ActiveRecord::Comments.comment(comment) do
```

Comment:
.comment is supposed to return what the yield returns, if it does not do that plz fix